addons/resource/models/resource_calendar.py

```diff
@@ -80,7 +80,7 @@ def default_get(self, fields):
     active = fields.Boolean("Active", default=True,
                             help="If the active field is set to false, it will allow you to hide the Working Time without removing it.")
     company_id = fields.Many2one(
-        'res.company', 'Company', domain=lambda self: [('id', 'in', [False, self.env.companies.ids])],
+        'res.company', 'Company', domain=lambda self: [('id', 'in', [False] + self.env.companies.ids)],
```

Comment:
this does not cause an issue, but I don't know any case where the False would be necessary, so we could have:

```suggestion
        'res.company', 'Company', domain=lambda self: [('id', 'in', self.env.companies.ids)],
```